frontend/apps/reader/modules/readerrolling.lua

```diff
@@ -457,6 +465,30 @@ You can set how many lines are shown.]])
         help_text = _([[When page overlap is enabled, some lines from the previous pages are shown on the next page.]]),
         sub_item_table = page_overlap_menu,
     }
+    local hide_nonlinear_text = _("When this option is enabled, if a document contains non-linear fragments, \z
+                                   they will be hidden from the normal page flow, \z
+                                   but they are still accessible through links, Toc or Go to. \z
+                                   This currently works only in single-page, non-scrolling mode.")
```

Comment:
I wanted to wrap it in the code, but not in the GUI. If there's some other way, I'd be happy to know (I don't know Lua, so I just googled for something).

Reply:
Oh, in that case you can use something like that
```
_([[multiline
string]])
```

Reply:
That leads to the linebreaks being retained in the in the string, which is what I'm trying to avoid:
![Screenshot_20201105_174502](https://user-images.githubusercontent.com/7943459/98270232-f586c780-1f8e-11eb-961e-0889944af6f0.png)

Reply:
Just make it a single long line in the code - we have already quite a few.